### DISCUSSION TITLE
whatever you do. dont update transmission on Mac OS Ventura. completely broken.

### DISCUSSION THREAD

**Discussion by irohzuko** [transmission/transmission#4938] [General]
leaving this here because It needs as much attention as ever, been using transmission for so many years and its been perfect but ever since upgrading, it is ok for the first minute but then it starts loosing peers and goes down to 0 and stops downloading, reset everything uninstalled reinstalled tried different torrents. my only fix was downgrading to v3, and it installed my torrent in 2 minutes

**Comment by pvr258** [comment]
Hi, I'm been having the exact same issue since upgrading more than a week ago. How can I reinstall V3 in an easy way? I only find links to multiple files I don't recognise... any help is appreciated.

**Reply by irohzuko** [reply]
yeah so, v3 now doesnt work so ive tried everything I even created a new user account on my Mac and installed it again new but nothing is working anymore. I switched to QbitTorrent now.

**Reply by pvr258** [reply]
I saw a developer mentioning that V4.01 would be released this Wednesday and that this issue might be tackled by it. Let's see :)

**Reply by pvr258** [reply]
I have no experience in using those. Where to find them? On the other hand, next version should be in less than 2 days

**Comment by Arnica2015** [comment]
I am having this very same issue, however some torrents complete way faster than others.  I restart the program and the speed returns to normal but after a minute it drops to zero.  Hopefully an update will help this.  I am also am wondering if my recent upgrade to wifi 6 could be causing the Transmission issue by somehow limiting peers.  irohzuko is QbitTorrent safe?  It asks me to over ride security issues.

**Reply by pvr258** [reply]
I don't think there is any link with wifi 6. I am not using wifi 6 and have the exact same issue you describe. Probably linked to Mac OS Ventura.

**Comment by ckerr** [comment]
FWIW `4.0.1` is scheduled for Wednesday.

...after it comes out I'm going to lock this discussion. If everything is *still* completely broken we'll have to start a new thread :tada:
